src/sinks/elasticsearch/mod.rs

```diff
@@ -172,12 +176,12 @@ impl ElasticsearchCommonMode {
     }
 }
 
-/// Configuration for api version.
+/// Configuration for Elasticsearch API version.
 #[configurable_component]
 #[derive(Clone, Debug, Eq, PartialEq)]
 #[serde(deny_unknown_fields, rename_all = "snake_case")]
 pub enum ElasticsearchApiVersion {
-    /// Auto-detect the api version. Will fail if endpoint isn't reachable.
+    /// Auto-detect the API version. Fails if the version endpoint (`/_cluster/state/version`) isn't reachable.
```

Comment:
Thoughts on linking to the elastic docs instead of writing it out? Just a thought, the documentation line just looked "long" especially compared to the other enum members.

Reply:
Haha @neuronull IDK if it works and more arbitrary design/prettiness things, but maybe line breaked?

```
Auto-detect the API version.

Fails if the [cluster state version endpoint](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-state.html#cluster-state-api-path-params) isn’t reachable.
```